include/zephyr/bluetooth/classic/hfp_hf.h

```diff
@@ -191,6 +191,19 @@ int bt_hfp_hf_register(struct bt_hfp_hf_cb *cb);
  */
 int bt_hfp_hf_send_cmd(struct bt_conn *conn, enum bt_hfp_hf_at_cmd cmd);
 
+#if defined(CONFIG_BT_HFP_HF_CLI)
```

Comment:
Please don't use #ifdefs in public header files. This prevents at least two important things:
 - Including the function as part of the documentation build (otherwise you need to declare an exception to the doxygen configuration)
 - Being able to use `if (IS_ENABLED(CONFIG_...)) {` in the code that calls the API.

Reply:
Updated.